server/resources/available.toggles

```diff
@@ -7,9 +7,9 @@
          "value": false
       },
       {
-         "key": "pipeline_config_single_page_app_key",
```

Comment:
Should we be removing this toggle? From what I remember we had this toggle since this feature is not complete and this gives the ability for admins to turn this off.